starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -49,4 +51,23 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  /**
+   * Test create method.
+   */  
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    Inventory savedInventory = this.inventoryDAO.create(inventory);
+
+    Query query = new Query();
+    query.addCriteria(Criteria.where("name").is("Amber"));
```

Comment:
Rather than doing a query here, you can do something like:` Assert.assertEquals(inventory.getName(), newInventory.getName())`

Reply:
Is it enough to just compare the inputs and outputs of the create function?